packages/link/CHANGELOG.md

```diff
@@ -0,0 +1,60 @@
+# @udecode/slate-plugins-link
```

Comment:
I don't think we need this old changelog. We should add a readme that says we uses the `@udecode/slate-plugins-link` as basis.